README.md

```diff
@@ -506,6 +514,17 @@ This allows to identify build errors caused by smalltalkCI updates and
 therefore helps to improve the service. It is possible to opt-out by using the
 `--no-tracking` option.
 
+### Pharo-specific Options
+
+Pharo added some specific options to customise its builds: 
+
+```yml
+env:
+  global:
+  - PHARO_VM=stable|latest              # choose stable/latest VM (default: stable)
```

Comment:
One more example to demonstrate why I'm trying to avoid dialect-specific env vars.
At some point, we want to be able to select latest VMs in Squeak and Moose as well, and then we end up with something like this:
```yml
env:
  global:
  - PHARO_VM=latest
  - SQUEAK_VM=latest
  - MOOSE_VM=latest
```

Instead, we could even support this:
```yml
smalltalk_vm: +latest
```
This would automatically select the right VM for the right image (Pharo, Squeak, Moose) with the `latest` option enabled, so basically the same as writing `Pharo-X+latest`,  `Squeak-X+latest`, and  `Moose-X+latest`.